src/winetricks

```diff
@@ -9803,6 +9821,41 @@ load_emu8086()
 
 #----------------------------------------------------------------
 
+w_metadata ev3 apps \
+    title="Lego Mindstorms EV3 Home Edition" \
+    publisher="Lego" \
+    year="2014" \
+    media="download" \
+    file1="LMS-EV3-WIN32-ENUS-01-01-01-full-setup.exe" \
+    installed_exe1="$W_PROGRAMS_X86_WIN/LEGO Software/LEGO MINDSTORMS EV3 Home Edition/MindstormsEV3.exe"
+
+load_ev3()
+{
+    if w_workaround_wine_bug 99999 "Installing vcrun2005 as Wine does not have MFC80.dll"
```

Comment:
Please file an actual bug for this. I'll be honest, it's unlikely to be fixed (a la mfc42, bug 657), but I'd still like to have a bug to track it.